docs/example-select.js

```diff
@@ -0,0 +1,19 @@
+function switchTab(event) {
+    let tab = event.target
```

Comment:
I guess `let` is widely enough supported now that it's fine for  us to use?

Reply:
Widely supported at this point https://caniuse.com/?search=let const will be the same